algrtm/src/components/OurWork.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import CustomLink from "./shared/CustomLink";
+import CustomIcon from "./shared/CustomIcon";
+
+const OurWork = () => {
+  return (
+    <section className=" w-full flex flex-col justify-center items-center text-center gap-6 mt-[2.75rem] mx-[1rem] md:mt-[7.75rem] md:mx-[4.56rem] xl:mx-[5.88rem] bg-neutralLight-neutral100 dark:bg-neutralDark-neutral100">
+      <h4 className="h4 lg:h2 text-neutralLight-neutral10 dark:text-neutralDark-neutral10">
+        Our work
+      </h4>
+      <div className="flex flex-col items-center justify-center w-full lg:flex-row gap-[1.5rem] md:flex-row xl:gap-[2rem] mt-[2rem]">
+        <div className=" flex flex-col justify-end items-center self-stretch pt-[9.29169rem] pb-[0.02081rem] rounded-[2rem] w-[21.44rem] text-neutralLight-neutral100 bg-[url('/heavyIndustry-bg-image.jpeg')]">
```

Comment:
This div and others with `bg-[some_url]` have shifted images and different sizes than on Figma. Adding such styling should fix this: `w-full md:w-[20.25rem] h-[25rem] bg-center bg-cover`